scss/_navbar-offcanvas.scss

```diff
@@ -66,15 +66,33 @@
       margin: 0;
       border: none;
     }
+    .input-group {
+      input.form-control {
+        height: 60px;
+        padding-left: 20px;
+      }
+      .input-group-append > button.btn-search {
```

Comment:
I was wondering what we were going to do with similar markup on the quickstart search box.  Thanks!